### PR TITLE
Add login help collapsible menu

### PR DESCRIPTION
**Pull request by zainab-amir**
- Added transition to collapsible menu
- Moved `forget password` and `other sign in issues` under menu

ezgif com-video-to-gif

[VAN-62](https://openedx.atlassian.net/browse/VAN-62)

### CODE REVIEW

File: src/logistration/tests/LoginPage.test.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should display login help button', () => {
+    const root = mount(reduxWrapper(<IntlLoginPage {...props} />));
+    expect(root.find('.field-link').text()).toEqual('Need help signing in?');
+  });
+
+  it('should have caret-right icon', () => {
```
Comment (waheedahmed) [inline src/logistration/tests/LoginPage.test.jsx:52]: I don't this test case is necessary.

File: src/logistration/tests/LoginPage.test.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+const IntlLoginPage = injectIntl(LoginPage);
+const mockStore = configureStore();
+const history = createMemoryHistory();
+
+describe('LoginPage', () => {
+  const initialState = {
+    logistration: {
+      forgotPassword: {
+        state: null
```
Comment (waheedahmed) [inline src/logistration/tests/LoginPage.test.jsx:22]: I think it's status.

**Review by waheedahmed — APPROVED** [review]
Couple of minor nits, otherwise 👍

File: src/logistration/tests/LoginPage.test.jsx (hunk 1 above)
Comment (zainab-amir) [reply src/logistration/tests/LoginPage.test.jsx:52]: will remove it

**Pull request merged**